### DISCUSSION TITLE
board_part not found

### DISCUSSION THREAD

**Discussion by ChenChung107360206** [bol-edu/HLS-SOC-Discussions#150] [Q&A]
我在跑source run_vivado.sh的時候，當執行到tcl檔的# set_property -name "board_part" -value "tul.com.tw:pynq-z2:part0:1.0" -objects $obj會出現ERROR
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/43467000/a02106ed-f18e-4d0b-80d0-0d587cc99638)
我有在網路上找方法嘗試解決，但這個ERROR消失後implement會fail。

想請問有沒有人也有遇到這個問題

**Comment by DanielDu9453** [comment]
有可能是你沒有匯入pynq-z2 的board file，匯入方法可以再https://github.com/bol-edu/caravel-soc_fpga 找到
